diff --git a/dist/index.js b/dist/index.js
index 9630a0308285a5891931f7e578e883984d427a2c..be7af0bb63fdf93e569c11e9169dd8cdb471956c 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,5 +1,5 @@
-import SuperDebug from './client/SuperDebug.svelte';
-export default SuperDebug;
+// import SuperDebug from './client/SuperDebug.svelte';
+// export default SuperDebug; // Not used and it breaks the Svelte 5 runes mode.
 export { SuperFormError, SchemaError } from './errors.js';
 // Everything from client/index.ts
 export { superForm, intProxy, numberProxy, booleanProxy, dateProxy, fieldProxy, formFieldProxy, stringProxy, arrayProxy, fileProxy, fileFieldProxy, filesProxy, filesFieldProxy, defaults, defaultValues, schemaShape, actionResult, superValidate, message, setMessage, setError, withFiles, removeFiles, fail } from './client/index.js';
